astro/src/pages/index.astro

```diff
@@ -50,7 +45,7 @@ import BaseLayout from "@layouts/Base.astro";
       </div>
       <button
         class="list-button text-white bg-[#2DA771] rounded py-2 px-4"
-        id="listing-button">List Your Stolen Bike</button
+        id="listing-button"><a href="/listing">List Your Stolen Bike</a></button
```

Comment:
a `a` tag inside a `button` is not a good practice.
you can have the `a` tag being displayed as a button by adding the same classes, and remove the `button` tag completely